agentops/client.py

```diff
@@ -58,27 +59,15 @@ def __init__(self, api_key: Optional[str] = None,
         self._session = None
         self._worker = None
         self._tags = tags
-        self.config = None
-
-        # These handle the case where .init() is used with optionals, and one of these
-        # params are None, which is does not trigger the Optional default in the constructor
-        if not api_key:
-            api_key = environ.get("AGENTOPS_API_KEY")
-            if not api_key:
-                logging.warning("AgentOps: No API key provided - no data will be recorded.")
-                return
-
-        if not parent_key:
-            parent_key = environ.get('AGENTOPS_PARENT_KEY', None)
-
-        if not endpoint:
-            endpoint = environ.get('AGENTOPS_API_ENDPOINT', 'https://api.agentops.ai')
-
-        self.config = Configuration(api_key,
-                                    parent_key,
-                                    endpoint,
-                                    max_wait_time,
-                                    max_queue_size)
+
+        try:
+            self.config = Configuration(api_key,
+                                        parent_key,
+                                        endpoint,
+                                        max_wait_time,
+                                        max_queue_size)
+        except ConfigurationError:
+            return
```

Comment:
You return but you aren't logging the error that resulted in the configuration error